Student/Student/Courses/CourseNavigation/CourseNavigationViewController.swift

```diff
@@ -39,47 +38,46 @@ class CourseNavigationTableViewController: UITableViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         configureTableView()
-        presenter.loadTabs()
-    }
-
-    func updateNavBar(title: String, backgroundColor: UIColor) {
-        navigationItem.title = title
-        navigationController?.navigationBar.useContextColor(backgroundColor)
+        presenter.viewIsReady()
```

Comment:
we probably want the nav bar color updated on viewWillAppear, just like we do with the AssignmentListViewController. We probably would need to store the color passed in from `updateNavBar`, or have the view reach into `courses.first.color`.

```
    override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)
        navigationController?.navigationBar.useContextColor(color)
    }
```